appserver/admin/admin-core/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.glassfish.main.admin</groupId>
         <artifactId>admin</artifactId>
-        <version>6.2.3-SNAPSHOT</version>
+        <version>6.2.4</version>
```

Comment:
Shouldn't it be rather 6.2.4-SNAPSHOT?

Reply:
... or 7.0.0-SNAPSHOT?

Reply:
... or do not change this in this very PR?

Reply:
> ... or do not change this in this very PR?

I expect it will be rebased after the release, so these changes will go away, but yeah, you are right.